README.md

```diff
@@ -118,13 +157,67 @@ ashDumpPersistenceSchemaOutputDirectoryPath := "~" // Changes directory
   - org.random.project.Data
 ```
 
-### 3. Serializer
+A `diff` command can be used to check the difference between the version of a schema from `develop`/`main` branch and the
+version from the current commit.
+
+![Easy to diff](docs/easy-to-diff.png)
+
+
+## Jackson Akka Serializer
+
+Using Jackson Serializer for Akka-persistence is also one of the pitfalls and this
+plugin provides an alternative by using a serializer that uses Circe.
+
+Dangerous code for Jackson:
+
+```scala
+case class Message(animal: Animal) extends MySer
+
+sealed trait Animal
+
+final case class Lion(name: String) extends Animal
+final case class Tiger(name: String) extends Animal
+```
+
+To run this code, a lot of Jackson annotations should be added:
```

Comment:
`To run this code` is a bit imprecise, this code _will run_ without these annotations ;)